cos-fleetshard-sync/src/main/kubernetes/kubernetes.yml

```diff
@@ -7,20 +7,17 @@ imagePullSecrets:
   - name: addon-pullsecret
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
namespaces as well should be added at ClusterRole

Reply:
done